### PR TITLE
Advance to closing time

### PR DESCRIPTION
**Pull request by james2m**
Added an advance_to_closing time method to do just that, plus working test.

### CODE REVIEW

File: lib/working_hours/computation.rb [hunk 1]
```diff
@@ -85,6 +85,26 @@ def advance_to_working_time time, config: nil
...
+      config ||= wh_config
+      time = in_config_zone(time, config: config).round
+      loop do
+        # skip holidays and weekends
+        while not working_day?(time, config: config)
+          time = (time + 1.day).beginning_of_day
+        end
+        # find next working range after time
+        time_in_day = time.seconds_since_midnight
+        time = time.beginning_of_day
```
Comment (jarthod) [inline lib/working_hours/computation.rb]: This won't work if there are several periods in a day, ex: `mon: {'08:00' => '12:00', '13:00' => '18:00'}`

Comment (james2m) [inline lib/working_hours/computation.rb]: Doesn't the test at line 177 cover this scenario? Or did I miss something? (I think I just repeated the same reply!). Can you tell me how it fails and I'll re-submit?

Comment (jarthod) [inline lib/working_hours/computation.rb]: Ah I see, you properly changed the return and increment so it's fine.
Can you please also add a test for the custom "midnight" handling (24:00) being represented as (23:59:59.99999)
So test with period ending at 24:00 and with current time being midnight or stuff like that.

**Comment by james2m** [comment]
I've pre-merged both my pull requests, so sending as a new one incorporating all your recommendations.

**Pull request closed without merging**